java-api-core/src/com/kinvey/java/store/BaseDataStore.java

```diff
@@ -188,7 +188,12 @@ public T find (@Nonnull String id) throws IOException {
                 return new ReadIdsRequest<>(cache, networkManager, this.storeType.readPolicy, ids).execute();
             }
         } else {
-            return new ReadIdsRequest<>(cache, networkManager, this.storeType.readPolicy, ids).execute();
+            if (storeType == StoreType.AUTO && deltaSetCachingEnabled) {
```

Comment:
Because before we had such conditions above in this code 
`if (storeType == StoreType.CACHE)  {`
`...`
     ` if (deltaSetCachingEnabled) {`
         `...`
      `}`
`...`
`}`
We checked `deltaSetCachingEnabled` only if `storeType == StoreType.CACHE` Now we change `StoreType.CACHE` to `StoreType.AUTO`